data/model/repo_mirror.py

```diff
@@ -52,7 +52,7 @@ def get_eligible_mirrors():
         (RepoMirrorConfig.sync_start_date <= now)
         & (RepoMirrorConfig.sync_retries_remaining > 0)
         & (RepoMirrorConfig.sync_status == RepoMirrorStatus.SYNCING)
-        & (RepoMirrorConfig.sync_expiration_date <= now)
+        & (RepoMirrorConfig.sync_expiration_date >= now)
         & (RepoMirrorConfig.is_enabled == True)
     )
```

Comment:
`sync_expiration_date <= now` is meant to find jobs that expired. If the expiration is in the future and it's in syncing state, it is not expired, right?

Reply:
Got it, I missed this. I am reverting this change.